autoscheduler/frontend/src/tests/Instructor.test.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
```

Comment:
Do we want this rule at all? From the related TSLint issue from [the typescript-eslint docs](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-var-requires.md):
```
ES2015-style imports are part of the JavaScript language specfication and recommended as the path going forward
```

Reply:
TypeScript won't let me use the bracket notation that lets me iterate through the object's non-nullable properties, so I just switched it to JavaScript. Hacky, I admit, but it's better than writing each test individually. Consequently, whatever compiler/engine Jest uses doesn't support ES2015-style imports in plain JS files, so I switched to the `require` syntax.

I looked into it more, and there is a workaround that I will implement in the next commit, but just a heads up, it requires disabling the "no-explicit-any" rule